src/conversations/Conversation.ts

```diff
@@ -360,6 +397,31 @@ export class ConversationV2 {
       error
     )
   }
+
+  export(): ConversationV2Export {
+    return {
+      version: 'v2',
+      topic: this.topic,
+      keyMaterial: Buffer.from(this.keyMaterial).toString('base64'),
```

Comment:
This is a part where my lack of knowledge scares me a bit. I'm assuming this is ok because clients would otherwise already have access to this keyMaterial when decrypting conversations locally? Should we emphasize Conversations MUST be stored in a secure location similar to the warning we have in the ConversationV2 constructor?

Reply:
That's a good point. Definitely deserves a section in the README that explains safe usage.